readme.md

```diff
@@ -486,6 +490,14 @@ OSARA also includes some other miscellaneous actions.
 - OSARA: Cycle automation mode of selected tracks
 - OSARA: Report global / Track Automation Mode
 - OSARA: Toggle global automation override between latch preview and off
+- Global automation override: All automation in latch mode
+- Global automation override: All automation in latch preview mode
+- Global automation override: All automation in read mode
+- Global automation override: All automation in touch mode
+- Global automation override: All automation in trim/read mode
+- Global automation override: All automation in write mode
+- Global automation override: Bypass all automation
+- Global automation override: No override (set automation modes per track)
```

Comment:
This section is for commands that OSARA introduces, as opposed to commands that are built into REAPER or are part of other extensions. These belong in one of the subsections of the "Supported REAPER and Extension Actions" section. I don't see one that immediately stands out as being a good fit, so it might be worth creating an "Automation" subsection or similar.

Reply:
How about renaming the envelopes section to Automation and putting them there?